src/OpenTelemetry/CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ Notes](../../RELEASENOTES.md).
   }
   ```
 
+* Moved the `InstrumentNameRegex` from `MeterProviderBuilderSdk` to `Guard`.
+  This regex is used to validate the view and instrument name. If you overwrite
+  the validation regex using reflection you have to update the code.
```

Comment:
On reflection we don't need these two sentences, as they're just an implementation detail. People can follow the PR link to see more detail.

```suggestion
```